README.md

```diff
@@ -68,6 +71,8 @@ Tool                  | Description
 [codecov]: https://codecov.io/gh/activeadmin/activeadmin
 [inch_badge]: http://inch-ci.org/github/activeadmin/activeadmin.svg?branch=master
 [inch]: http://inch-ci.org/github/activeadmin/activeadmin
+[tidelift_badge]: https://tidelift.com/badges/github/activeadmin/activeadmin
+[tidelift]: https://tidelift.com/subscription/pkg/rubygems-activeadmin?utm_source=rubygems-arbre&utm_medium=readme
```

Comment:
Should this be `utm_source=rubygems-activeadmin` instead?

Reply:
😱